### DISCUSSION TITLE
What is the difference between the tacotron2 models?

### DISCUSSION THREAD

**Discussion by KyuubiYoru** [coqui-ai/TTS#1910] [General Q&A]
With the tacotron2 models what does the DCA, DDC, DDC_ph stand for?

**Comment by king-dahmanus** [comment]
ddc is double decoder consistancy, and I discourage using it to train. Based on the pretrained ljspeech tacotron2 ddc model it skips things a lot. Don't know why they recommended using it in the first place, same for capacitron, with its instability. Is it fixed?

**Reply by KyuubiYoru** [reply]
This sounds like, max_decoder_steps is set to low, I thing they change the default from 500 to 10000, but probably not for the Pretrained models.
Changing the the value in the config of the Pretrained model should work.

**Comment by erogol** [comment]
DDC -> Double decoder consistency -> https://coqui.ai/blog/tts/solving-attention-problems-of-tts-models-with-double-decoder-consistency
DCA -> Dynamic Conv Attention -> https://arxiv.org/pdf/1910.10288.pdf

Go with DDC if you have enough VRAM else use DCA
